### PR TITLE
fix: Update to hydra helm chart to version 1.7.4

### PR DESCRIPTION
**Pull request by jfelten**
## Proposed changes

Updated to hydra 1.7.4 - Change init postgres container to migrate db with latest version
Also updated schema for ingress

## Checklist

- [x] I have read the [contributing guidelines](../blob/master/CONTRIBUTING.md)
- [x] I have read the [security policy](../security/policy)
- [x] I confirm that this pull request does not address a security
      vulnerability. If this pull request addresses a security vulnerability, I
      confirm that I got green light (please contact
      [security@ory.sh](mailto:security@ory.sh)) from the maintainers to push
      the changes.
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation within the code base (if appropriate)

## Further comments
Tested and verified working.

### CODE REVIEW

File: helm/charts/hydra/templates/deployment.yaml [hunk 1]
```diff
@@ -42,7 +42,7 @@ spec:
           image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
           imagePullPolicy: {{ .Values.image.pullPolicy }}
           command: ["hydra"]
-          args: ["migrate", "sql", "-e", "--yes", "--config", "/etc/config/config.yaml"]
+          args: ["migrate", "sql", "--yes", "{{ .Values.hydra.config.dsn }}"]
```
Comment (aeneasr) [inline helm/charts/hydra/templates/deployment.yaml:45]: Please revert this change unless there is a good reason. It will break DSN configuration in many scenarios.

File: helm/charts/hydra/Chart.yaml [hunk 2]
```diff
@@ -1,8 +1,8 @@
 apiVersion: v2
-appVersion: "v1.4.6"
+appVersion: "v1.7.4"
 description: A Helm chart for deploying ORY Hydra in Kubernetes
 name: hydra
-version: 0.0.1
+version: 0.0.2
```
Comment (aeneasr) [inline helm/charts/hydra/Chart.yaml:5]: This is changed automatically in the release process and should be reverted:

```suggestion
version: 0.0.1
```

**Review by aeneasr — COMMENTED** [review]
Thank you, this looks good but please revert the `deployment.yml` change - thank you! :)

File: helm/charts/hydra/templates/deployment.yaml (hunk 1 above)
Comment (jfelten) [reply helm/charts/hydra/templates/deployment.yaml:45]: The reason for the change is that the syntax of the migrate command changed with v1.7.4. The v.1.7.4 init container using the old syntax cause an error when I tested and is the main reason for this PR. I will gladly revert if this is not true. See [postgres example](https://www.ory.sh/hydra/docs/configure-deploy)

Comment (aeneasr) [reply helm/charts/hydra/templates/deployment.yaml:45]: Are you sure? Here is the help message from v1.7.4:

```
hydra help migrate sql
Run this command on a fresh SQL installation and when you upgrade Hydra to a new minor version. For example,
upgrading Hydra 0.7.0 to 0.8.0 requires running this command.

It is recommended to run this command close to the SQL instance (e.g. same subnet) instead of over the public internet.
This decreases risk of failure and decreases time required.

You can read in the database URL using the -e flag, for example:
	export DSN=...
	hydra migrate sql -e

### WARNING ###

Before running this command on an existing database, create a back up!

Usage:
  hydra migrate sql <database-url> [flags]

Flags:
  -h, --help            help for sql
  -e, --read-from-env   If set, reads the database connection string from the environment variable DSN or config file key dsn.
  -y, --yes             If set all confirmation requests are accepted without user interaction.

Global Flags:
      --config string     Config file (default is $HOME/.hydra.yaml)
      --skip-tls-verify   Foolishly accept TLS certificates signed by unkown certificate authorities
foobar@foobars-MacBook-Pro ~ % hydra version
Version:    v1.7.4
Git Hash:   ff980e6d6c5a5f17b7a0c55e7593a9ec75ea76ef
Build Time: 2020-08-31T15:38:54Z
```

Comment (jfelten) [reply helm/charts/hydra/templates/deployment.yaml]: looks like I'm wrong, and must have conflated the error with something else. I verified the migrate runs with the existing commands and will revert the change:

Output:
```
The following migration is planned:

Version          Name                                              Status    
20190100000001   client                                            Applied   
20190100000001   client                                            Applied   
20190100000002   client                                            Applied   
20190100000002   client                                            Applied   
20190100000003   client                                            Applied   
20190100000003   client                                            Applied   
20190100000004   client                                            Applied   
20190100000004   client                                            Applied   
20190100000004   client                                            Applied   
20190100000005   client                                            Applied   
20190100000005   client                                            Applied   
20190100000006   client                                            Applied   
20190100000006   client                                            Applied   
20190100000007   client                                            Applied   
20190100000007   client                                            Applied   
20190100000008   client                                            Applied   
20190100000008   client                                            Applied   
20190100000008   client                                            Applied   
20190100000009   client                                            Applied   
20190100000009   client                                            Applied   
20190100000009   client                                            Applied   
20190100000010   client                                            Applied   
20190100000010   client                                            Applied   
20190100000011   client                                            Applied   
20190100000011   client                                            Applied   
20190100000011   client                                            Applied   
20190100000012   client                                            Applied   
20190100000012   client                                            Applied   
20190100000013   client                                            Applied   
20190100000013   client                                            Applied   
20190100000013   client                                            Applied   
20190100000014   client                                            Applied   
20190100000014   client                                            Applied   
20190100000014   client                                            Applied   
20190200000001   jwk                                               Applied   
20190200000001   jwk                                               Applied   
20190200000002   jwk                                               Applied   
20190200000002   jwk                                               Applied   
20190200000003   jwk                                               Applied   
20190200000003   jwk                                               Applied   
20190200000004   jwk                                               Applied   
20190200000004   jwk                                               Applied   
20190200000004   jwk                                               Applied   
20190300000001   consent                                           Applied   
20190300000001   consent                                           Applied   
20190300000002   consent                                           Applied   
20190300000002   consent                                           Applied   
20190300000003   consent                                           Applied   
20190300000003   consent                                           Applied   
20190300000004   consent                                           Applied   
20190300000004   consent                                           Applied   
20190300000004   consent                                           Applied   
20190300000005   consent                                           Applied   
20190300000005   consent                                           Applied   
20190300000005   consent                                           Applied   
20190300000006   consent                                           Applied   
20190300000006   consent                                           Applied   
20190300000006   consent                                           Applied   
20190300000007   consent                                           Applied   
20190300000007   consent                                           Applied   
20190300000007   consent                                           Applied   
20190300000008   consent                                           Applied   
20190300000008   consent                                           Applied   
20190300000008   consent                                           Applied   
20190300000009   consent                                           Applied   
20190300000009   consent                                           Applied   
20190300000010   consent                                           Applied   
20190300000010   consent                                           Applied   
20190300000010   consent                                           Applied   
20190300000011   consent                                           Applied   
20190300000011   consent                                           Applied   
20190300000012   consent                                           Applied   
20190300000012   consent                                           Applied   
20190300000012   consent                                           Applied   
20190300000013   consent                                           Applied   
20190300000013   consent                                           Applied   
20190300000013   consent                                           Applied   
20190300000014   consent                                           Applied   
20190300000014   consent                                           Applied   
20190400000001   oauth2                                            Applied   
20190400000001   oauth2                                            Applied   
20190400000002   oauth2                                            Applied   
20190400000002   oauth2                                            Applied   
20190400000003   oauth2                                            Applied   
20190400000003   oauth2                                            Applied   
20190400000004   oauth2                                            Applied   
20190400000004   oauth2                                            Applied   
20190400000005   oauth2                                            Applied   
20190400000005   oauth2                                            Applied   
20190400000005   oauth2                                            Applied   
20190400000006   oauth2                                            Applied   
20190400000006   oauth2                                            Applied   
20190400000006   oauth2                                            Applied   
20190400000007   oauth2                                            Applied   
20190400000007   oauth2                                            Applied   
20190400000007   oauth2                                            Applied   
20190400000008   oauth2                                            Applied   
20190400000008   oauth2                                            Applied   
20190400000009   oauth2                                            Applied   
20190400000009   oauth2                                            Applied   
20190400000009   oauth2                                            Applied   
20190400000010   oauth2                                            Applied   
20190400000010   oauth2                                            Applied   
20190400000010   oauth2                                            Applied   
20190400000011   oauth2                                            Applied   
20190400000011   oauth2                                            Applied   
20200521071434   consent                                           Applied   
20200521071434   consent                                           Applied   
20200521071434   consent                                           Applied   
20200527215731   client                                            Applied   
20200527215732   client                                            Applied   
20200819163013   add_client_id_subject_idx_to_access_and_refresh   Applied   
20200819163013   add_client_id_subject_idx_to_access_and_refresh   Applied   
20200819163013   add_client_id_subject_idx_to_access_and_refresh   Applied   
[POP] 2020/09/24 17:50:58 info - Migrations already up to date, nothing to apply
[POP] 2020/09/24 17:50:58 info - 0.0477 seconds
Successfully applied migrations!
```

**Comment by aeneasr** [comment]
Awesome, thank you for putting in the work and sorry for my late review!
